api_bonus.yaml

```diff
@@ -0,0 +1,144 @@
+            type: object
+            properties:
+              id:
+                type: string
+                description: The identifier of the created request.
+          headers:
+            Location:
+              type: string
+              description: |-
+                Location (URL) of created request resource.
+                A GET request to this URL returns the request status and details.
+          examples:
+            application/json:
+              id: 01BX9NSMKVXXS5PSP2FATZMYYY
+
+    get:
+      operationId: getBonusEligibilityCheck
+      summary: Get eligibility (ISEE) check information for user's bonus
+      responses:
+        '202':
+          description: Processing request.
+          schema:
+            type: object
+            properties:
+              id:
+                type: string
+                description: The identifier of the created request.
+          examples:
+            application/json:
+              id: 01BX9NSMKVXXS5PSP2FATZMYYY
+        '200':
+          description: Request processed.
+          schema:
+            type: object
+            properties:
+              id:
+                type: string
+                description: The identifier of the created request.
+          examples:
+            application/json:
+              # REQUEST (isee): NO_DATA, ELIGIBLE, INELIGIBLE, NOT_FOUND
+              status: ELIGIBLE
+              members:
+                - name: "Mario"
+                  surname: "Rossi"
+                - name: "Piero"
+                  surname: "Rossi"
+                - name: "Giulia"
+                  surname: "Rossi"
+              max_amount: 50000
+              tax_benefit: 3000
+
+    delete:
+      operationId: cancelBonusEligibilityProcedure
+      summary: Cancel bonus eligibility check procedure (avoids sending a push notification when done)
+      responses:
+        '200':
+          description: Request canceled.
+          schema:
+            type: object
+            properties:
+              id:
+                type: string
+                description: The identifier of the canceled request.
+          examples:
+            application/json:
+              id: 01BX9NSMKVXXS5PSP2FATZMYYY
+
+  "/bonus/vacanze":
+    post:
+      operationId: startBonusActivationProcedure
+      summary: Start bonus activation procedure
+      responses:
+        '202':
+          description: Request accepted.
+          schema:
+            type: object
+            properties:
+              id:
+                type: string
+                description: The identifier of the created request.
+          headers:
+            Location:
+              type: string
+              description: |-
+                Location (URL) of created activation resource.
+                A GET request to this URL returns the request status and details.
+          examples:
+            application/json:
+              id: 01BX9NSMKVXXS5PSP2FATZMYYY
+
+    get:
```

Comment:
this should have the activate request ID in the path

Reply:
not if the activate request id is the user's fiscal code. can we have more than one activation request for each user ?